cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/ConnectorMockTest.java

```diff
@@ -178,7 +178,7 @@ public void beforeEach(TestInfo testInfo, VertxTestContext testContext) {
                 new DefaultKafkaAgentClientProvider(),
                 metricsProvider,
                 new DefaultZooKeeperAdminProvider(),
-                pfa, 10_000);
+                pfa, 10_000, new BrokersInUseCheck());
```

Comment:
Maybe you can fix the lines -> one item per line? It seems to be slightly broken already before, but no need to break it even more.

Reply:
See previous comment, it's not in place anymore and replaced by mocking the admin client.